src/wled/wled.py

```diff
@@ -590,7 +590,7 @@ async def upgrade(self, *, version: str | AwesomeVersion) -> None:
             msg = "Unexpected upgrade error; No session or device"
             raise WLEDUpgradeError(msg)
 
-        if self._device.info.architecture not in {
+        if self._device.info.architecture.lower() not in {
```

Comment:
Let's adjust the model to downcase it (so we don't end up downcasing it on every use).

../Frenck

Reply:
Done